tests/api_connexion/schemas/test_dag_run_schema.py

```diff
@@ -0,0 +1,171 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import unittest
+
+from airflow.api_connexion.schemas.dag_run_schema import (
+    DAGRunCollection, dagrun_collection_schema, dagrun_schema,
+)
+from airflow.models import DagRun
+from airflow.utils import timezone
+from airflow.utils.session import provide_session
+from airflow.utils.types import DagRunType
+from tests.test_utils.db import clear_db_runs
+
+
+class TestDAGRunBase(unittest.TestCase):
+
+    def setUp(self) -> None:
+        clear_db_runs()
+        self.now = timezone.utcnow()
+
+    def tearDown(self) -> None:
+        clear_db_runs()
+
+
+class TestDAGRunSchema(TestDAGRunBase):
+
+    @provide_session
+    def test_serialzie(self, session):
+        dagrun_model = DagRun(run_id='my-dag-run',
+                              run_type=DagRunType.MANUAL.value,
+                              execution_date=self.now,
+                              start_date=self.now,
+                              conf='{"start": "stop"}'
+                              )
+        session.add(dagrun_model)
+        session.commit()
+        dagrun_model = session.query(DagRun).first()
+        deserialized_dagrun = dagrun_schema.dump(dagrun_model)
+
+        self.assertEqual(
+            deserialized_dagrun[0],
+            {
+                'dag_id': None,
+                'dag_run_id': 'my-dag-run',
+                'end_date': None,
+                'state': 'running',
+                'execution_date': str(self.now.isoformat()),
+                'external_trigger': True,
+                'start_date': str(self.now.isoformat()),
+                'conf': '{"start": "stop"}'
```

Comment:
Why do we have a string here? The specification define object here.

Reply:
I found that this can solve it for us without any validation code from our side:

`app.add_api("openapi.yaml", strict_validation=True)`

https://connexion.readthedocs.io/en/latest/request.html#parameter-validation

Should I add it?